sqlhelper.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+#database setup function
+def setup_database(dbfile):
```

Comment:
```suggestion
def setup_database(dbfile: str):
```
we want to make sure we only receive strings for this parameter, it will prevent a sqlite error later on

add this to every time we take in dbfile as a parameter